DotnetMakeDeb/Deb/DebPackage.cs

```diff
@@ -468,6 +494,10 @@ public void WritePackage(Stream outStream)
 			{
 				AddFile(postRmFileName, 0, 0, 493 /* 0755 */, postRmIsText);
 			}
+			foreach (var controlFileItem in controlFileItems)
+			{
+				AddFile(controlFileItem.SourcePath, 0, 0, 493 /* 0755 */, controlFileItem.IsText);

```

Comment:
And I'm not sure about this executable file mode. Is it necessary or recommended for all control files? The default mode is 644.

Reply:
I have read the documentation, since the `control` file is actually a `sh`/`bash` script file it is indeed need to be executable.

Reply:
Also the template file?

Reply:
I am not really sure with template file needed to be in executable mode or not.
I also have not tested if the template file is not in executable mode will the script running properly or not.

Reply:
Now it should no longer become a problem because user can define their own `Mode` for the control file.